automation/notebooks-table-data.csv

```diff
@@ -0,0 +1,2 @@
+display_name, notebook_name, youtube_video_path, github_repository_path, arxiv_index
```

Comment:
what would be the purpose of the columns `youtube_video_path`, `github_repository_path`?

Reply:
The initial purpose of `youtube_video_path` was to add some complementary tutorial, but I see that it is not necessary. The one I do see necessary is the `github_repository_path`, this in case you want to add the source code of the paper you are covering.

Reply:
nice